### ISSUE TITLE
Suggestion: Add texture exporting

### ISSUE THREAD

**Issue by mariomadproductions** [KillzXGaming/Smash-Forge#7]
This would be useful to have, rather than having to rely on the BMS scripts and/or TexConv2.exe (and Photoshop with the DDS plugin if you want them in PNG).

**Comment by ScanMountGoat** [comment]
Newer versions of Forge can export OpenGL textures as png. @KillzXGaming

**Comment by KillzXGaming** [comment]
Yeah currently only have that for Switch (BNTX) but i'll add in support for FTEX soon.
